src/app/models/evt-models.ts

```diff
@@ -331,3 +331,163 @@ export class Word extends GenericElement {
+    altIdentifier?: AltIdentifier;
+    msName?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when msName is handled
+    country?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when country is handled
+    region?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when region is handled
+}
+
+export class MsContents extends GenericElement {
+    summary?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when summary is handled
+    msItem?: MsItem;
+    msItemStruct?: MsItemStruct;
+}
+
+export class PhysDesc extends GenericElement {
+    objectDesc?: ObjectDesc;
+    bindingDesc?: BindingDesc;
+    decoDesc?: DecoDesc;
+    handDesc?: HandDesc;
+    accMat?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when accMat is handled
+    additions?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when additions is handled
+    musicNotation?: MusicNotation;
+    scriptDesc?: ScriptDesc;
+    sealDesc?: SealDesc;
+    typeDesc?: TypeDesc;
+}
+
+export class MsPart extends GenericElement {
+    msIdentifier: MsIdentifier;
+    msContents?: MsContents;
+    physDesc?: PhysDesc;
+    msPart?: MsPart;
+    additional?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when additional is handled
+    history?: History;
+    head?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when head is handled
+}
+
+export class History extends GenericElement {
+    acquisition?: Acquisition;
+    origin?: Origin;
+    provenance?: Provenance;
+    summary?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when summary is handled
+}
+
+export class AltIdentifier extends GenericElement {
+    noteEl?: Note[];
+    idno?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when idno is handled
+    collection?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when collection is handled
+    repository?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when repository is handled
+    region?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when region is handled
+    settlement?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when settlement is handled
+}
+
+export class MsItemStruct extends GenericElement {
+    n?: string;
+    class?: string;
+    defective?: boolean;
+    locus?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when locus is handled
+    author?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when author is handled
+    title?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when title is handled
+    rubric?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when rubric is handled
+    finalRubric?: Array<ParseResult<GenericElement>>; // TODO: Add specific type finalRubric defective is handled
+    bibl?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when bibl is handled
+}
```

Comment:
According to the TEI guidelines there are more elements that can be used to descrive the msItemStruct (see https://www.tei-c.org/release/doc/tei-p5-doc/en/html/MS.html#mscoit - 10.6.1 The and Elements)